packages/experiments/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@wordpress/dependency-injection",
```

Comment:
I have just noticed that it should be `@wordpress/experiments` before we start using it.